snmp/datadog_checks/snmp/data/conf.yaml.example

```diff
@@ -104,9 +109,8 @@ instances:
 
     ## @param community_string - string - optional
     ## Only useful for SNMP v1 & v2.
-    ## Enclose the community string with single quote like below (to avoid special character being interpreted).
     #
-    # community_string: 'public'
+    # community_string: public
```

Comment:
Could you keep this recent change from https://github.com/DataDog/integrations-core/commit/55093a8715d7c3b0cef5f1707753bd78d61a26dc